packages/shell-evaluator/src/shell-evaluator.ts

```diff
@@ -43,7 +43,7 @@ class ShellEvaluator<EvaluationResultType = ShellResult> {
       shellApi[cmd]?.acceptsRawInput &&
       !(argv[0] ?? '').startsWith('(')
     ) {
-      return shellApi[cmd](input);
+      return shellApi[cmd](input.trim().split(/\s+/).slice(1).join(' '));
```

Comment:
This is still a bit buggy, now multiple adjacent whitespaces in the command are converted into a single whitespace